.github/scripts/with-commit.sh

```diff
@@ -0,0 +1,4 @@
+git config --local user.name "GitHub Action"
```

Comment:
Maybe change this to 'Pantheon Automation' / bot@getpantheon.com which other projects within pantheon use for this: https://github.com/pantheon-upstreams/wordpress-composer-managed/commits/main 
https://github.com/pantheon-systems/wordpress-composer-managed/blob/default/devops/scripts/deploy-public-upstream.sh#L51-L54

Funky things in the Github UI can happen when commits are made by a real email that we don't own.